client/src/components/Workflow/Editor/Forms/FormTool.vue

```diff
@@ -1,32 +1,37 @@
 <template>
     <ToolCard
-        v-if="hasData"
-        :id="configForm.id"
-        :version="configForm.version"
-        :title="configForm.name"
-        :description="configForm.description"
+        :id="configForm?.id || ''"
```

Comment:
I get the feeling that it might be cleaner to have a separate component for a missing tool ?

Reply:
I initially thought the same, but it requires more changes to maintain inputs and headers. The components also need refactoring due to the complex logic and props. We can do it on follow-up for the next release.